kubernetes/loculus/templates/_common-metadata.tpl

```diff
@@ -371,8 +371,11 @@ organisms:
       {{- end }}
       {{- end }}
     referenceGenomes:
-      {{ $referenceGenomes:= include "loculus.generateReferenceGenome" $instance.referenceGenomes | fromYaml }}
-      {{ $referenceGenomes | toYaml |nindent 8}}
+      {{- range $subOrganismName, $referenceGenome := $instance.referenceGenomes }}
+      {{ $subOrganismName }}:
```

Comment:
`subOrganism` or `suborganism`? In most of the other places, we now have `suborganism` / `Suborganism`.